src/RepoAuditor/ExecuteModules.py

```diff
@@ -91,12 +91,15 @@ def CalcResultInfo(
                 elif eval_info.module.name in ignore_warnings_module_names:
                     continue
 
-            if result == EvaluateResult.Error:
+            elif result == EvaluateResult.Error:
                 return -1, "errors were encountered"
 
-            if result == EvaluateResult.Warning:
+            elif result == EvaluateResult.Warning:
                 return_code = 1
 
+            elif result == EvaluateResult.DoesNotApply:
+                return_code = 2
+
     return return_code, "" if return_code == 0 else "warnings were encountered"

```

Comment:
This should probably be updated to account for `DoesNotApply`.

Reply:
I updated the function docstring to state that `DoesNotApply` equates to a success.